doc/publishing-api-syntactic-usage.md

```diff
@@ -91,15 +91,15 @@ Requests to update an existing draft content item:
   - `type` the type of unpublishing to create/perform.
 
 ### Optional request params:
-  - `explanation` (optional) Message to display on page for `gone`,
+  - `explanation` Message to display on page for `gone`,
```

Comment:
Yep, I've done another pass over this section, which has brought this information back  		fe2af28.